src/views/Correction.vue

```diff
@@ -893,6 +928,12 @@ h2 {
   font-size: 1.125rem;
 }
 
+.detail-comment {
+  ::v-deep(.placeholder) {
+    color: $gray7;
+  }
+}
```

Comment:
I bet you don't need this block, since you already do this on lines 904-906.

Reply:
oh i named the selector wrong good catch, its for detail-comment-input